apps/api/src/app/environments-v1/usecases/create-environment/create-environment.usecase.ts

```diff
@@ -67,6 +97,21 @@ export class CreateEnvironment {
       });
     }
 
+    await this.createNovuIntegrationsUsecase.execute(
+      CreateNovuIntegrationsCommand.create({
+        environmentId: environment._id,
+        organizationId: environment._organizationId,
+        userId: command.userId,
+      })
+    );
+
     return environment;
   }
+
+  private getEnvironmentColor(name: string, commandColor?: string): string | undefined {
+    if (name === EnvironmentEnum.DEVELOPMENT) return '#ff8547';
+    if (name === EnvironmentEnum.PRODUCTION) return '#7e52f4';
```

Comment:
Default env colors, client will also fallback for legacy environments

Reply:
should we consider moving this const to Shared?